lms/djangoapps/teams/static/teams/js/views/team_card.js

```diff
@@ -19,22 +19,36 @@
                 '<span class="member-count"><%= membership_message %></span>' +
                 '<ul class="list-member-thumbs"></ul>'
             ),
+            profileImageTemplate: _.template(
```

Comment:
See my comment above. This feels to me like breaking up rendering logic along lines which are not really defined by the rendering boundaries themselves. I mostly used `_.template()` here as a shortcut to string interpolation.